processor/transformprocessor/README.md

```diff
@@ -14,44 +14,131 @@
-> This documentation applies only to version `0.120.0` and later. For information on earlier versions, please refer to the previous [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/release/0.119.x/processor/transformprocessor/README.md).
+> This documentation applies only to version `0.120.0` and later. **Configuration from previous version is still supported**, but no longer documented in this README. For information on earlier versions, please refer to the previous [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/release/0.119.x/processor/transformprocessor/README.md).
 
-The transform processor modifies telemetry based on configuration using the [OpenTelemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl).
+The Transform Processor modifies telemetry based on configuration using the [OpenTelemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl) (OTTL).
 
-For each signal type, the processor takes a list of conditions and statements associated to a [Context type](#contexts) and executes the conditions and statements against the incoming telemetry in the order specified in the config.
-Each condition and statement can access and transform telemetry using functions and allow the use of a condition to help decide whether the function should be executed.
+For each signal type, the processor takes a list of statements and executes statements against the incoming telemetry in the order specified in the config.
+Each statement can access and transform telemetry using functions and allow the use of a condition to help decide whether the function should be executed.
 
 - [Config](#config)
+- [Writing OTTL](#writing-ottl)
 - [Grammar](#grammar)
-- [Contexts](#contexts)
 - [Supported functions](#supported-functions)
 - [Examples](#examples)
 - [Troubleshooting](#troubleshooting)
 - [Contributing](#contributing)
+- [Feature Gate](#feature-gate)
 
 ## Config
 
-The transform processor allows configuring multiple context statements for traces, metrics, and logs.
-The value of `context` specifies which [OTTL Context](#contexts) to use when interpreting the associated statements, and can be either explicitly set or [inferred](#context-inference).
-The global conditions and statement strings, which must be OTTL compatible, will be passed to OTTL and interpreted using the associated context.
-The condition string should contain a Where clause body without the `where` keyword at the beginning.
+The Transform Processor allows configuring OTTL statements for traces, metrics, and logs.
 
-Each context will be processed in the order specified.
-Within a context, each global condition is checked and if any evaluates to true, the statements are executed in order.
-If a context doesn't meet any of the conditions, then the associated statement will be skipped.
+> [!NOTE]
```

Comment:
This note is equals and visually very close to the below note, should we remove this one?

Reply:
I can't see which note you're referring to anymore, but I would like to keep it at the top of the 2 config styles.

Reply:
I'd like to keep this note as a warning to anyone who read through `Writing OTTL` that this is something they can skip unless they are interested.